alerter.cpp

```diff
@@ -26,6 +31,7 @@ void alertInCelcius(float farenheit) {
 int main() {
     alertInCelcius(400.5);
     alertInCelcius(303.6);
+    assert(alertFailureCount == 1);

```

Comment:
This assert will pass when either one of the above calls increments the `alertFailureCount`. Strengthen the test to make it clear which one should pass and which one not.